plugins/azure/virtualmachines/diskUnattachedAndDefaultEncryption.js

```diff
@@ -0,0 +1,51 @@
+var async = require('async');
+
+var helpers = require('../../../helpers/azure/');
+
+module.exports = {
+    title: 'Disk Volumes Unattached and Default Encryption',
+    category: 'Virtual Machines',
+    domain: 'Compute',
+    description: 'Ensures that no default encrypted Azure virtual machine disks are in unattached state.',
+    more_info: 'Encrypting virtual machine disk volumes helps protect and safeguard your data to meet organizational security and compliance commitments.When a virtual machine (VM) in Azure is deleted, by default, any disks that are attached to the VM aren\'t deleted.',
```

Comment:
```suggestion
    more_info: 'Encrypting virtual machine disk volumes helps protect and safeguard your data to meet organizational security and compliance commitments. When a virtual machine (VM) in Azure is deleted, by default, any disks that are attached to the VM aren\'t deleted.',
```

Reply:
Can we make this more meaningful. Like having an unattached disk with default encryption can lead to data leakage